### ISSUE TITLE
Passing only one bam file to the programm

### ISSUE THREAD

**Issue by asan-emirsaleh** [ISUgenomics/SequelTools#15]
Hello!
I have _FILENAME_.bam.1 file downloaded from ncbi.
How can I use it in SequelTools CLI? I am going to rename it (to left an '.1' suffix off) and to create a subreads.txt file with the only record pointing to that file. Then I will pass it to SequelTolls. Is it correct?

**Comment by aseetharam** [comment]
Yes, @asantat, that should work. When you download the file using tools like `wget` or `curl`, it automatically renames the files with the `.1` extension, if there is already a file with that name, perhaps that's what caused this in the first place. Hope this helps!
